debian/control

```diff
@@ -14,6 +17,7 @@ Build-Depends:
  python-testtools,
  python-mock,
  python-nose,
+ help2man
```

Comment:
minor "hint":  I usually keep even trailing ',' so that next diff (if you add more entries) becomes smaller (less chance of dsc patches to break etc)